src/Calendar/components/DayGrid/utils/useTotalEventsToShow.js

```diff
@@ -16,8 +16,8 @@ const useTotalEventsToShow = () => {
   // even if there are no events show right away
   const [totalEventsToShow, setTotalEventsToShow] = useState(0);
 
-  console.log('rowHeight: ', rowHeight);
-  console.log('rowRef: ', rowRef);
+  // console.log('rowHeight: ', rowHeight);
```

Comment:
I'm good if you just remove all these instead of comment them out

Reply:
Fixed in 938971c